sky/engine/core/compositing/SceneBuilder.idl

```diff
@@ -14,6 +14,7 @@
   void pop();
 
   void addPicture(Offset offset, Picture picture, Rect bounds);
+  void addStatistics(unsigned long enabledOptions, Rect bounds);
```

Comment:
For the text, pretty much :/ But for the graphs, they will resize to fill the available bounds. Another PR incoming for that.